code/threshold_canny.ipynb

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
<p>Great stuff. I will get back to you on what should be better about the end result but in general <code>edges_img</code>  looks fairly similar to what we need for the next step (i.e. chain code)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/markusait/lcmd/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>